src/ebonite/client/base.py

```diff
@@ -153,31 +153,35 @@ def get_environment(self, name: str) -> RuntimeEnvironment:
         """
         return self.meta_repo.get_environment_by_name(name)
 
-    def run_service(self, name: str, image: Image, environment: RuntimeEnvironment = None, **kwargs) -> RuntimeInstance:
+    def run_service(self, name: str, image: Image, environment: RuntimeEnvironment = None,
+                    service_type: str = 'docker', **kwargs) -> RuntimeInstance:
         """
         Runs model service and stores it to repository
 
         :param name: name of instance to run
         :param image: image to run instance from
         :param environment: environment to run instance in, if no given `localhost` is used
+        :param service_type: type of service to be run, default is regular Docker container
         :return: :class:`~ebonite.core.objects.RuntimeInstance` instance representing run instance
         """
-        env_name = 'localhost'
+        from ebonite.build import RunnerBase, RunnersCatalog
+        runner: RunnerBase = RunnersCatalog.get(service_type)
+
+        env_name = f'{service_type}_localhost'
```

Comment:
doesn't feel right to create and push default env in `run_service` client method. but i dont have a right way to do it in mind yet. i'll write when i got one (if i got one)

Reply:
For me it's OK but both of us are obviously subjective. My motivation to do this is described above.